app/Account.php

```diff
@@ -300,7 +344,7 @@ public function canDowngrade()
      */
     public function isSubscribed()
     {
-        if ($this->has_access_to_paid_version_for_free) {
+        if ($this->has_access_to_paid_version_for_free == 1) {
```

Comment:
Why not just "return $this->has_access_to_paid_version_for_free == 1"

Reply:
Damn that was a test. I'll put it back.